tests/test_segments.py

```diff
@@ -914,7 +925,7 @@ def test_workspaces(self):
 				{'contents': '3: w3', 'highlight_groups': ['w_urgent', 'w_visible', 'workspace']},
 				{'contents': '4: w4', 'highlight_groups': ['w_focused', 'w_urgent', 'w_visible', 'workspace']},
 			])
-			self.assertEqual(i3wm.workspaces(pl=pl, segment_info=segment_info, only_show=['visible'], strip=3), [
+			self.assertEqual(i3wm.workspaces(pl=pl, segment_info=segment_info, only_show=['visible'], strip=True), [
```

Comment:
Though since `workspaces` was deprecated I expected it to have *only* old functionality.